test/e2e/validation_test.go

```diff
@@ -309,11 +309,8 @@ func (tc *testContext) sshSetup() error {
 // PowerShell, on the host specified by the provided IP.
 func (tc *testContext) runPowerShellSSHJob(name, command, ip string) (string, error) {
 	// Modify command to work when default shell is the newer Powershell version present on Windows Server 2022.
-	// We only test Windows Server 2022 if a custom VXLAN port is used (i.e. vSphere)
 	powershellDefaultCommand := command
-	if tc.hasCustomVXLAN {
-		powershellDefaultCommand = strings.ReplaceAll(command, "\\\"", "\"")
-	}
+	powershellDefaultCommand = strings.ReplaceAll(command, "\\\"", "\"")
```

Comment:
Won't this cause the AWS tests fails on this PR since they still use 2019? I think `if tc.hasCustomVXLAN || tc.CloudProvider.GetType() == config.AzurePlatformType ` is the needed clause for this to merge

Reply:
@saifshaikh48  is right, until https://github.com/openshift/windows-machine-config-operator/pull/1091 merges AWS still needs this. I proposed an alternative platform-specific [approach](https://github.com/openshift/windows-machine-config-operator/pull/1091/commits/8ea8b220b8bb76b2dce0d1857abf13d3d05b48f9).